data/records.json

```
@@ -125,4 +125,25 @@
         "license": "freeware",
         "upload_date": "12/11/2021"
     }
+    {
+        "name": "SpotiBot - Spotify Automation",
+        "description_short": "The bot makes automatic login in spotify, mutes ads and tries to skip them",
+        "description": "SpotiBot is a automation for muting and skipping ads. You can use it as template to try to create your own spotify automation. Makes automatically login before starts to muting",
+        "tags": ["Spotify", "automation","WebBot", "web","bot"],
+        "programming_language": "python",
+        "technology": ["botcity web framework"],
+        "OS": ["windows"],
+        "thumbnail_url": "https://imgur.com/a/Iys0UG4",

```

This need to be the embedded URL:
```suggestion
        "thumbnail_url": "https://i.imgur.com/kBarjAq.png",

```